package.json

```diff
@@ -6,7 +6,7 @@
   "main": "src/index.js",
   "scripts": {
     "lint": "eslint src",
-    "start": "run-p start:**",
+    "start": "source ./.env && run-p start:**",
```

Comment:
I'm getting an error here when running `npm start` with `zsh` and `bash` saying that `source` doesn't exist. Maybe we should put it in its own bash file which sources `.env` and execute hugo/webpack from there? Or something like `./with_env.sh hugo  ...` which just exports the environment variables and then executes the arguments

Reply:
ugh...

Reply:
is the error that `source` doesn't exist or `.env` ?

Reply:
do you think this will work correctly?

```bash
#!/bin/bash

ENV_FILE=.env

if test -f "$ENV_FILE"; then
    source $ENV_FILE
fi
```

Reply:
I realized we should probably test for existence of the file anyway

Reply:
@aliceriot Sorry, it's `sh: 1: source: not found`